### DISCUSSION TITLE
Any handy tips on building

### DISCUSSION THREAD

**Discussion by r3stl355** [delta-io/delta-rs#1040] [General]
Came across this library and wanted to test but the basic build is failing with few errors (e.g. `the name `tls` is defined multiple times` 
 or `function or associated item not found in hyper_rustls::HttpsConnector<_>`) I just did a pull and `cargo build` (tried both v 1.66 and 1.64 toolchain - same errors). Any suggestions?
Also, docs on getting started are next to none, would there be an interest in extending readme by adding more info on how to start using this library with few explicit examples perhaps, I'm happy to do that

**Comment by wjones127** [comment]
Sorry there are several difference packages in the project, and their default features aren't compatible right now (due to conflicting TLS implementations). For now I'd recommend building the specific package you care about. e.g. `cd rust && cargo build`.

We have a few usage examples in the Rust docs: https://docs.rs/deltalake/0.6.0/deltalake/. We would welcome enhancements, though. And if you want to add some to the readme that seems fine too.
